Sources/OktaOidc/iOS/OktaOidc+BrowserIOS.swift

```diff
@@ -19,9 +19,23 @@ extension OktaOidc: OktaOidcBrowserProtocolIOS {
 
     @objc public func signInWithBrowser(from presenter: UIViewController,
                                         callback: @escaping ((OktaOidcStateManager?, Error?) -> Void)) {
+        signInWithBrowser(from: presenter, payload: [:], callback: callback)
+    }
+    
+    @objc public func signInWithBrowser(from presenter: UIViewController,
```

Comment:
**Note:** The original `signInWithBrowser(from:callback:)` method still works, so that this does't introduce a breaking API change.